src/vs/editor/browser/widget/multiDiffEditorWidget/multiDiffEditorWidget.ts

```diff
@@ -45,6 +45,10 @@ export class MultiDiffEditorWidget extends Disposable {
 		this._register(recomputeInitiallyAndOnChange(this._widgetImpl));
 	}
 
+	public reveal(revealData: IMultiDiffEditorOptionRevealData): void {
```

Comment:
I would inline `IMultiDiffEditorOptionRevealData`, because it is just fields and not a complicated interface.
That makes it easier to understand the signature.